README.md

```diff
@@ -118,4 +118,26 @@ graph LR
     iadd-->enc_sa;
 
     softmax--"Probablity"-->pred;
-```
\ No newline at end of file
+```
+
+
+## The Folder Structure
+The folder of the project is desigened by slight structure for expansibility and convenience:
+```shell
+.
+├── data # mounted data and scripts
```

Comment:
The comment does not seem necessary, consider remove.

Reply:
Done